<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_supervisor_customer</class>
 <widget class="QMainWindow" name="MainWindow_supervisor_customer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>345</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>660</width>
    <height>345</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>660</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Nest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      拉黑</string>
    </property>
   </widget>
   <widget class="QListWidget" name="customerPass">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 12pt &quot;Calibri&quot;;
border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color:rgba(0,0,0,0);</string>
    </property>
   </widget>
   <widget class="QListWidget" name="customerUnpass">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>161</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: italic 12pt &quot;Calibri&quot;;
border:2px groove gray;border-radius:10px;padding:2px 4px;
background-color:rgba(0,0,0,0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      退出登录</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>客户黑名单</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="title">
     <string>顾客详情：</string>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>订单数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="ordernumber">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>投诉数：</string>
     </property>
    </widget>
    <widget class="QLabel" name="badjudgement">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      查看详情</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;华文楷体&quot;;</string>
    </property>
    <property name="text">
     <string>可预订的客户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>230</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>      查看详情</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>230</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>       恢复</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
border-image: url(:/images/button1.png);
font: 12pt &quot;华文楷体&quot;;
color:rgb(0, 0, 0)
}
QPushButton:hover
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(102, 102, 102)
}
QPushButton:pressed
{
border-image: url(:/images/button1.png);
font: 12pt  &quot;华文楷体&quot;;
color:rgb(180, 180, 180)
}
</string>
    </property>
    <property name="text">
     <string>     酒店管理</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
